style/client.css

```diff
@@ -99,6 +99,17 @@ hr {
 	border-right: 0;
 }
 
+button.button.disabled,
+button.button.disabled:hover,
+button.button.disabled:active {
```

Comment:
We might also need to add `.mainmenu ` versions of these selectors so they're the right specificity.